src/pages/api/hello.ts

```diff
@@ -1,14 +1,21 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { Result } from "antd";
+import { responsiveArray } from "antd/lib/_util/responsiveObserve";
 import type { NextApiRequest, NextApiResponse } from "next";
 import { getDateInfo } from "../../../src/lib/backend/database-utils";
 type Data = {
   name: string;
 };
 
-getDateInfo();
-export default function handler(
-  req: NextApiRequest,
-  res: NextApiResponse<Data>
-) {
-  res.status(200).json({ name: "John Doe" });
-}
+const handler = async (req: NextApiRequest, res: NextApiResponse<Data>) => {
```

Comment:
We should move to next-connect (nc) as it has some useful features, including automatic error handling.